services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/controller/DpmController.java

```diff
@@ -45,7 +45,7 @@ public InternalDPMResultList createNetworkConfiguration(@RequestBody NetworkConf
         return dpmService.createNetworkConfiguration(networkConfiguration);
     }
 
-    @PutMapping({"/network-configuration", "v4/network-configuration"})
+    @PostMapping({"/network-configuration", "v4/network-configuration"})
     @DurationStatistics
     public InternalDPMResultList updateNetworkConfiguration(@RequestBody NetworkConfiguration networkConfiguration) throws Exception {
         checkNetworkConfiguration(networkConfiguration);
```

Comment:
@songxiaoyan  and @xieus This change will break dpm. There are two PostMapping with the same url after you change line48 from Put to Post.